bin/topological-inventory-inventory-upload

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+
+lib = File.expand_path("../lib", __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+STDOUT.sync = true
+
+require "bundler/setup"
+require "topological_inventory/sync/inventory_upload_worker"
+
+def parse_args
+  require 'optimist'
+  opts = Optimist.options do
+    opt :queue_host, "Hostname of the Platform's kafka queue", :type => :string,
+        :default => ENV["QUEUE_HOST"] || "localhost"
```

Comment:
Minor, but I think it's preferable to do `ENV.fetch("QUEUE_HOST", "localhost")`

Reply:
Okay done, we do this literally everywhere so we have lots to change then